addons/skin.estuary/xml/Custom_1109_TopBarOverlay.xml

```diff
@@ -16,48 +16,51 @@
 				<height>170</height>
 				<texture>frame/osdfade.png</texture>
 			</control>
-			<control type="image">
-				<left>15</left>
-				<top>10</top>
-				<width>300</width>
-				<height>140</height>
-				<texture>$VAR[PlayerClearLogoVar]</texture>
-				<aspectratio aligny="center" align="center">keep</aspectratio>
-			</control>
-			<control type="textbox">
-				<label>$VAR[NowPlayingBreadcrumbsVar]</label>
-				<font>font45</font>
-				<shadowcolor>text_shadow</shadowcolor>
-				<top>0</top>
-				<height>150</height>
-				<left>330</left>
-				<right>400</right>
-				<aligny>center</aligny>
-				<visible>!String.IsEmpty(Player.Art(tvshow.clearlogo))</visible>
-			</control>
 			<control type="group">
-				<visible>!Window.IsActive(pvrosdchannels) + !Window.IsActive(pvrchannelguide)</visible>
-				<visible>String.IsEmpty(Player.Art(clearlogo))</visible>
-				<visible>String.IsEmpty(Player.Art(tvshow.clearlogo))</visible>
-				<animation effect="fade" time="150">VisibleChange</animation>
-				<left>20</left>
-				<right>400</right>
-				<control type="label">
+				<animation effect="slide" end="90,0" time="0" condition="Skin.HasSetting(touchmode)">conditional</animation>
```

Comment:
The animation does not work for me, the Back button overlaps with the Top Bar info

![image](https://user-images.githubusercontent.com/5781142/85923369-4a4c6000-b882-11ea-8215-f2ff54ff2144.png)

This is fixed for me by `<animation effect="slide" end="90,0" time="0" condition="Skin.HasSetting(touchmode) + [Window.IsActive(videoosd) | Window.IsActive(musicosd)]">conditional</animation>`